6MI0800239/pract.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+void readFile(ifstream& ifs,int& sum,int& proiz)
+{
+    ifstream file("result.txt");

```

Comment:
You have the file already given as an argument but you open another one that you never close :thinking: